src/custom/utils/signatures.ts

```diff
@@ -13,9 +13,7 @@ import { ChainId } from '@uniswap/sdk'
 
 import { GP_SETTLEMENT_CONTRACT_ADDRESS } from 'constants/index'
 import { TypedDataDomain, Signer } from 'ethers'
-import { registerOnWindow } from './misc'
-
-export { OrderKind } from '@gnosis.pm/gp-v2-contracts'
```

Comment:
It wasn't used here at all?
No linter complaints? 

:scream:

Reply:
it was just exported, not used